src/logging.cc

```diff
@@ -36,8 +36,8 @@ Logger::Logger(const char* file, int line, Severity severity, LogStream* stream)
   std::strftime(time_val, sizeof(time_val), "%m%d %H:%M:%S ", &local_time);
   (*this) << kSeverities_[severity]
           << time_val
-          << std::hex << std::this_thread::get_id() << std::dec << " "
-          << file_ << ":" << line_ << " ";
+          << std::dec << std::this_thread::get_id() << std::dec << " "
```

Comment:
I misunderstood `std::thread::id` so `std::dec` won't work, but you can change this line to:
`static_cast<unsigned int>(hasher(std::this_thread::get_id())) << " "`
and add this above line 37:
`std::hash<std::thread::id> hasher;`

Reply:
Done. Thanks for pointing it out, my stdlib-fu is quite rusty these days 😐